src/qcodes/instrument_drivers/Keithley/Keithley_2450.py

```diff
@@ -306,14 +319,16 @@ def __init__(self, parent: "Keithley2450", name: str, proper_function: str) -> N
             "the internal reference measurements (autozero) of the"
             "instrument.",
         )
+        """This command enables or disables automatic updates tothe internal reference measurements (autozero) of theinstrument."""
```

Comment:
```suggestion
        """This command enables or disables automatic updates to the internal reference measurements (autozero) of the instrument."""
```

Reply:
Do this manually to also resolve the docstr argument